src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -2065,7 +2146,8 @@ TEST(DirectTaskTransportTest, TestNoWorkerRequestedIfStealingUnavailable) {
   uint32_t max_tasks_in_flight_per_worker = 10;
   CoreWorkerDirectTaskSubmitter submitter(
       address, raylet_client, client_pool, nullptr, lease_policy, store, task_finisher,
-      NodeID::Nil(), kLongTimeout, actor_creator, max_tasks_in_flight_per_worker);
+      NodeID::Nil(), kLongTimeout, actor_creator, max_tasks_in_flight_per_worker,
+      absl::nullopt, 1);
```

Comment:
Is it possible to add a test that heartbeat reports the backlog (not sure if it's covered above).